src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java

```diff
@@ -239,7 +239,7 @@ private boolean isNodeInOntology(OWLClass currentTreeNode, T currentTerm, HashSe
 		boolean condition5 = !config.getIgnoreEntitiesWithChebiXref();
 		boolean condition6 = !hasChebiXref(currentTerm);
 
-		return ((condition1 && condition2) || (condition3 && condition4)) && (condition5 || condition6);
+		return ((condition1 && condition2 && !condition3) || (condition3 && condition4)) && (condition5 || condition6);
```

Comment:
Why is negated now?

Reply:
@abecerra its not negated its added to the condition.

Reply:
If we apply the IRI prefix filter we don’t want terms to load if they match the default namespace but don’t have the correct prefix. ie. Passing condition1 and condition2 is not sufficient, checkIRIPrefix must be null